3_navigation.ipynb

```diff
@@ -64,7 +117,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's check the information by turning the check on and off in \"Displays\" panel. The meaning of each item is as follows:\n",
```

Comment:
"Let's check the information by turning the checks on and off in the \"Displays\" panel. The meaning of each item is as follows:\n",